src/screens/TokenSwapReview.js

```diff
@@ -98,15 +93,15 @@ const TokenSwapReview = () => {
     });
   };
 
-  const onButtonPress = () => {
+  const onSwapButtonPress = () => {
     const pinParams = {
       cb: executeSend,
       canCancel: true,
       screenText: t`Enter your 6-digit pin to authorize operation`,
       biometryText: t`Authorize operation`,
       biometryLoadingText: t`Building transaction`,
     };
-    navigation.navigate('PinScreen', pinParams);
+    NavigationService.navigate('PinScreen', pinParams);
```

Comment:
chore: I'd recommend agains using the NavigationService for common navigations: for that it's better that we use the `navigation` obtained from `useNavigation()` as it was before.

The only use for it is to ask the app itself to reset to main and other navigations that the screen needs to ask for the global Navigation object to do in its behalf.